NSURLSessionCancellation/ViewController.swift

```diff
@@ -12,69 +12,69 @@ class ViewController: UIViewController {
 
     // A sample URL that takes a while to download (so that cancellaton can be demonstrated)
     
-    private let url = NSURL(string: "http://yahoo.com")!
+    fileprivate let url = URL(string: "http://yahoo.com")!
```

Comment:
`private` is more restrictive, `file private` has the same rules as the old private so i let the migrator do its thing here. 

In this case - a demo class - I think either is fine but using `fileprivate` means the member variable can be accessed by subclasses and extensions (common case for Swift) too whereas `private` would restrict that. 

I can make the change or we can leave it as is since its a demo class?